equistore-core/include/equistore.hpp

```diff
@@ -1294,7 +1318,7 @@ class TensorBlock final {
         return array;
     }
 
-    friend class TensorMap;
+    friend class equistore_torch::TensorBlockHolder;
```

Comment:
I don't understand the change here. Is this correct?

* doesn't this require to have equistore-torch also installed?
* why is the friend class TensorMap? Isn't TensorMap still needed as friend class?